### PR TITLE
Adds Login.gov content where CSPs are listed

### LINKED ISSUE #32643: Update login content to call out new Login.gov CSP
## Issue Description
Content needs to be updated to reflect the new Login.gov CSP when it gets enabled on VA.gov.  Currently, the content reads as **Sign in with your DS Logon, My HealtheVet, or ID.me account** and it needs to be updated to read as **Sign in with your Login.gov, ID.me, DS Logon, and My HealtheVet.**

In a quick search of the vets-website repo the content exists in SignInDescription, Confirmation pages (multiple), AuthApp, UnverifiedPrefillAlert, and SignIn components.  Try to look for additional places the content might be.

---
## Tasks
- [x] Update the Sign In content to read in the same order as the buttons. (Order: Login.gov, ID.me, DS Logon, MHV)
- [x] Search for any other places on vets-website for the 3 CSPs (DS Logon, MHV, ID.me) and add Login.gov.

## Acceptance Criteria
- [x] The content is gated behind the `loginGov` feature flag until Login.gov launch
- [x] When Login.gov feature flag is enabled I can see the CSP order as Login.gov, ID.me, DS Logon, and MHV.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by asg5704**
## Description
This PR changes content throughout vets-website to ensure Login.gov is featured with the other Service Providers: ID.me, DS Logon, and My HealtheVet.  The content is gated behind the `login_gov` feature flag.

## Original issue(s)
Closes department-of-veterans-affairs/va.gov-team#32643

## Testing done
Unit testing / visual

## Screenshots
Examples: 
Screen Shot 2021-11-30 at 11 15 38 AM
Screen Shot 2021-11-30 at 11 13 48 AM

## Acceptance criteria
- [ ] With the `login_gov` feature flag enabled, I can see Login.gov is added to the list of CSPs
- [ ] With the `login_gov` feature flag disabled, I cannot see Login.gov in the list of CSPs 

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/auth/containers/AuthApp.jsx [hunk 1]
```diff
@@ -216,8 +217,9 @@ export class AuthApp extends React.Component {
             <p>
               Please try again, and this time, select “Accept” on the final page
               of the identity verification process. Or, if you don’t want to
-              verify your identity with ID.me, you can try signing in with your
-              premium DS Logon or premium My HealtheVet username and password.
+              verify your identity with Login.gov or ID.me, you can try signing
```
Comment (Kwoly) [inline src/applications/auth/containers/AuthApp.jsx:220]: Are we fine with showing Login.gov in situations like this even if the `login_gov` flag is disabled?

File: src/applications/static-pages/cta-widget/tests/index.unit.spec.js [hunk 2]
```diff
@@ -3,10 +3,67 @@ import React from 'react';
...
+      ...mviStatus,
+    },
+    fetchMHVAccount: sinon.spy(),
+  },
+  mockStore: {
+    getState: () => ({
+      featureToggles: {
+        loading: false,
+      },
+      // login: {
```
Comment (Kwoly) [inline src/applications/static-pages/cta-widget/tests/index.unit.spec.js:44]: probably remove this if it's unused

**Comment by AGarcia-Clarity** [comment]
@Kwoly we're going to wait to convert this from a Draft PR. The reason being that the content might change in the next week or so.

**Review by Kwoly — APPROVED** [review]

File: src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js [hunk 3]
```diff
@@ -28,7 +29,7 @@ export const AuthContent = ({
     <div>
       <div itemScope itemType="http://schema.org/Question">
         <h2 itemProp="name" id="how-can-the-va-prescription-re">
-          How can VA's prescription tools help me manage my health care?
+          How can VA’s prescription tools help me manage my health care?
```
Comment (timwright12) [inline src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js:32]: <3

**Review by timwright12 — APPROVED** [review]
Nothing jumping out at me here, looks good

File: src/applications/personalization/profile/tests/e2e/direct-deposit/non-verified-idme.cypress.spec.js [hunk 4]
```diff
@@ -97,6 +97,26 @@ describe('Direct Deposit', () => {
 
         notIDme2FAAlertShown();
 
+        directDepositAPIsNotCalled();
+      });
+    },
+  );
+  context(
+    'when user has 2FA set up but signed in with MHV/My HealtheVet',
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/direct-deposit/non-verified-idme.cypress.spec.js:105]: This context description is wrong. Please delete all changes to this file as we _don't_ want to treat Login.gov users as MHV or DSLogon users with regards to Direct Deposit.

**Review by erikphansen — CHANGES_REQUESTED** [review]
please revert the additions to the Direct Deposit Cypress tests

File: src/applications/personalization/profile/tests/e2e/direct-deposit/non-verified-idme.cypress.spec.js (hunk 4 above)
Comment (AGarcia-Clarity) [reply src/applications/personalization/profile/tests/e2e/direct-deposit/non-verified-idme.cypress.spec.js]: Reverted back.

**Review by erikphansen — APPROVED** [review]

**Comment by AGarcia-Clarity** [comment]
:wave: Could I get review from the following teams @department-of-veterans-affairs/vsa-public-websites-frontend 🙏 

Thank you 🚀

File: src/applications/static-pages/cta-widget/components/messages/SignIn.jsx [hunk 5]
```diff
@@ -14,9 +18,8 @@ const SignIn = ({
     headerLevel,
     alertText: (
       <p>
-        Try signing in with your <strong>DS Logon</strong>,
-        <strong>My HealtheVet</strong>, or <strong>ID.me</strong> account. If
-        you don’t have any of those accounts, you can create one now.
+        Sign in with your existing <ServiceProvidersText isBold /> account.{' '}
```
Comment (kelsonic) [inline src/applications/static-pages/cta-widget/components/messages/SignIn.jsx:21]: We may want to run this past content? Sorry for the pings if it already has been 🙂 CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/UnauthContent/index.js [hunk 6]
```diff
@@ -60,25 +61,7 @@ export const UnauthContent = () => (
...
-        milConnect), <strong>or</strong>
-      </li>
-      <li>
-        A verified <strong>ID.me</strong> account that you can{' '}
-        <a href="https://api.id.me/en/registration/new">
-          create here on VA.gov
-        </a>
-      </li>
-    </ul>
+    <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/UnauthContent/index.js:64]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/AuthContent/index.js [hunk 7]
```diff
@@ -68,25 +69,7 @@ export const AuthContent = ({ cernerFacilities, otherFacilities }) => (
...
-        milConnect), <strong>or</strong>
-      </li>
-      <li>
-        A verified <strong>ID.me</strong> account that you can{' '}
-        <a href="https://api.id.me/en/registration/new">
-          create here on VA.gov
-        </a>
-      </li>
-    </ul>
+    <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/AuthContent/index.js:72]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/UnauthContent/index.js [hunk 8]
```diff
@@ -59,28 +60,7 @@ export const UnauthContent = () => (
...
-        milConnect), <strong>or</strong>
-      </li>
-      <li>
-        A verified <strong>ID.me</strong> account that you can{' '}
-        <a href="https://api.id.me/en/registration/new">
-          create here on VA.gov
-        </a>
-      </li>
-    </ul>
+    <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/UnauthContent/index.js:63]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/AuthContent/index.js [hunk 9]
```diff
@@ -99,31 +100,7 @@ export const AuthContent = ({
...
-                  A verified <strong>ID.me</strong> account that you can{' '}
-                  <a
-                    rel="noreferrer noopener"
-                    href="https://id.me/en/registration/new"
-                  >
-                    create here on VA.gov
-                  </a>
-                </li>
-              </ul>
+              <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/AuthContent/index.js:103]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/UnauthContent/index.js [hunk 10]
```diff
@@ -54,25 +55,7 @@ export const UnauthContent = () => (
...
-        milConnect), <strong>or</strong>
-      </li>
-      <li>
-        A verified <strong>ID.me</strong> account that you can{' '}
-        <a href="https://api.id.me/en/registration/new">
-          create here on VA.gov
-        </a>
-      </li>
-    </ul>
+    <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/UnauthContent/index.js:58]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js [hunk 11]
```diff
@@ -108,28 +109,7 @@ export const AuthContent = ({
...
-                  eBenefits and milConnect), <strong>or</strong>
-                </li>
-                <li>
-                  A verified <strong>ID.me</strong> account that you can{' '}
-                  <a href="https://api.id.me/en/registration/new">
-                    create here on VA.gov
-                  </a>
-                </li>
-              </ul>
+              <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/AuthContent/index.js:112]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/UnauthContent/index.js [hunk 12]
```diff
@@ -40,25 +41,7 @@ export const UnauthContent = () => (
...
-        milConnect), <strong>or</strong>
-      </li>
-      <li>
-        A verified <strong>ID.me</strong> account that you can{' '}
-        <a href="https://api.id.me/en/registration/new">
-          create here on VA.gov
-        </a>
-      </li>
-    </ul>
+    <ServiceProvidersList />
```
Comment (kelsonic) [inline src/applications/static-pages/health-care-manage-benefits/view-test-and-lab-results-page/components/UnauthContent/index.js:44]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/medical-copays-cta/components/App/index.js [hunk 13]
```diff
@@ -24,10 +28,8 @@ export const App = ({ loggedIn, show, toggleLoginModal }) => {
         <p>With this tool, you can:</p>
       ) : (
         <p>
-          Try signing in with your <strong>DS Logon</strong>,{' '}
-          <strong>My HealtheVet</strong>, or <strong>ID.me</strong> account. If
-          you don’t have any of those accounts, you can create one now. When you
-          sign in or create an account, you’ll be able to:
+          Sign in with your existing <ServiceProvidersText isBold /> account.{' '}
```
Comment (kelsonic) [inline src/applications/static-pages/medical-copays-cta/components/App/index.js:31]: CC @RLHecht @DanielleThierryUSDSVA

File: src/applications/static-pages/cta-widget/components/messages/SignIn.jsx (hunk 5 above)
Comment (kelsonic) [reply src/applications/static-pages/cta-widget/components/messages/SignIn.jsx:21]: image

This is what will be in place of the `<ServiceProvidersList>` fysa CC @RLHecht @DanielleThierryUSDSVA

Comment (AGarcia-Clarity) [reply src/applications/static-pages/cta-widget/components/messages/SignIn.jsx:21]: Thank you @kelsonic 👍 .  There is no rush on this I think the Login.gov launch is being pushed back a bit.

Comment (kelsonic) [reply src/applications/static-pages/cta-widget/components/messages/SignIn.jsx:21]: Related slack thread: https://dsva.slack.com/archives/C52CL1PKQ/p1638553654346700

File: src/applications/static-pages/medical-copays-cta/components/App/index.js (hunk 13 above)
Comment (RLHecht) [reply src/applications/static-pages/medical-copays-cta/components/App/index.js:31]: We should use contractions in the sentence: If you don't have one of these accounts, you'll be able to create a free...

Comment (RLHecht) [reply src/applications/static-pages/medical-copays-cta/components/App/index.js:31]: Actually: If you don't have one of these account, you can create a free... (change **you'll be able to** to **can**

**Comment by AGarcia-Clarity** [comment]
@kelsonic Are we good to go now?

**Review by kelsonic — APPROVED** [review]
lgtm, nice work! 🚀 thanks for looping in @RLHecht and @DanielleThierryUSDSVA with these changes! 🎉

**Comment by AGarcia-Clarity** [comment]
@department-of-veterans-affairs/vsa-ebenefits-frontend @department-of-veterans-affairs/vsa-bam-1-frontend @department-of-veterans-affairs/vsa-debt-frontend @department-of-veterans-affairs/vsa-caregiver-frontend 

A review would be greatly appreciated

**Comment by AGarcia-Clarity** [comment]
@department-of-veterans-affairs/vsa-ebenefits-frontend @department-of-veterans-affairs/vsa-bam-1-frontend @department-of-veterans-affairs/vsa-debt-frontend @department-of-veterans-affairs/vsa-caregiver-frontend 

A review would be greatly appreciated

**Review by Mottie — APPROVED** [review]
This PR doesn't change any of the code my team supports.

**Review by jerekshoe — APPROVED** [review]

**Review by craigwheeler — APPROVED** [review]

**Pull request merged**